__tests__/app.test.js

```diff
@@ -162,3 +162,79 @@ describe("GET /api/articles/:article_id/comments", () => {
 
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+	test('should return a status code of 201 and a new comment posted to the database', () => {
+		const newComment = {
+			username: 'icellusedkars',
+			body: 'Great article on how to build endpoints!'
+		}		
+		
+		return request(app)
+		.post('/api/articles/6/comments')
+		.send(newComment)
+		.expect(201)
+		.then(({ body }) => {
+			expect(body.comment).toBe('Great article on how to build endpoints!')
+		})
+	});
+	
+	test('should return a status code of 400 when user leaves a blank comment', () => {
+		const newComment = {
+			username: 'icellusedkars',
+			body: ''
+		}		
+		
+		return request(app)
+		.post('/api/articles/6/comments')
+		.send(newComment)
+		.expect(400)
+		.then(({ body }) => {
+			expect(body.msg).toBe('Bad Request')
+		})
+	});
+	
+	test('should return a status code of 400 for an invalid article_id', () => {
+		const newComment = {
+			username: 'icellusedkars',
+			body: 'Very useful article !'
+		}		
+		
+		return request(app)
+		.post('/api/articles/rubbish/comments')
+		.send(newComment)
+		.expect(400)
+		.then(({ body }) => {
+			expect(body.msg).toBe('Bad Request')
+		})
+	});
+
+	test('should return a status code of 404 for an article_id which doesn"t exist', () => {
+		const newComment = {
+			username: 'icellusedkars',
+			body: 'Very useful article !'
+		}		
+		
+		return request(app)
+		.post('/api/articles/99/comments')
+		.send(newComment)
+		.expect(404)
+		.then(({ body }) => {
+			expect(body.msg).toBe('Not Found')
+		})
+	});
+
+	test('should return a status code of 404 for a username which doesn"t exist', () => {
+		const newComment = {
+			username: 'idonotexist',
+			body: 'Boring article :('
+		}		
+		
+		return request(app)
+		.post('/api/articles/6/comments')
+		.send(newComment)
+		.expect(404)
+		.then(({ body }) => {
+			expect(body.msg).toBe('Username Not Found')
+		})
+	});
+});
```

Comment:
Lovely testing so far! I'd like to see a couple more tests:

201 - ignores unnecessary properties on the sent comment
400 - missing required fields i.e. no username/no body properties